CRT.c

```diff
@@ -803,8 +803,21 @@ int CRT_scrollWheelVAmount = 10;
 ColorScheme CRT_colorScheme = COLORSCHEME_DEFAULT;
 
 ATTR_NORETURN
-static void CRT_handleSIGTERM(ATTR_UNUSED int sgn) {
+static void CRT_handleSIGTERM(int sgn) {
+   const char* signal_str;
+
    CRT_done();
+
+   if (!CRT_crashSettings->changed)
+      _exit(0);
+
+   signal_str = strsignal(sgn);
```

Comment:
```suggestion
   CRT_done();

   if (!CRT_crashSettings->changed)
      _exit(0);

   const char* signal_str = strsignal(sgn);
```

Reply:
Personally I prefer declarations at the top. We do not want this for `htop` code?

Reply:
For htop it is preferred to have declarations initialized with a value when sensibly possible. As C99 allows for declarations to be mixed in where they are needed it's usually preferred to place them where they are first needed. Thus when there's the choice between having uninitialized declarations for variables at the top and C99-style intermixed declarations with a valid assignment, then prefer C99. Overall the scope and lifetime of values should be minimized (sensibly) and uninitialized variables should be avoided.